src/lib/components/dashboard/credentials/CredentialCard.svelte

```diff
@@ -55,11 +54,14 @@
     hoverEffect = false;
   }
 
-  const removeCredentialHandler = async () => {
-    await removeCredential(credential.credentialId);
-    const responseJson = await fetchCredentialsByFolder($selectedFolder.id);
-    const response = await sendMessage("decryptMeta", responseJson.data);
-    credentialStore.set(response.data);
+  const triggerMoreActions = (e: any) => {
+    showMoreOptions.set(true);
+    buttonRef.set(e.currentTarget);
```

Comment:
first set buttonref then call showmore, otherwise there will be race on where to show showMoreOptions.